routing/route.hpp

```diff
@@ -72,12 +72,13 @@ class Route
   double GetCurrentDistanceToEndMeters() const;
   double GetMercatorDistanceFromBegin() const;
 
-  void GetCurrentTurn(double & distanceToTurnMeters, turns::TurnItem & turn) const;
+  bool GetCurrentTurn(double & distanceToTurnMeters, turns::TurnItem & turn) const;
   /// @return true if GetNextTurn() returns a valid result in parameters, false otherwise.
-  /// \param distanceToTurnMeters is a distance from current possition to the second turn.
+  /// \param distanceToTurnMeters is a distance from current position to the second turn.
```

Comment:
Of course not ) Previous version took TurnItem by reference whereas current version tooks a vector of TurnItem by reference and "returns a valid result in parameters" is no more valid.